SoftLayer/managers/vs.py

```diff
@@ -252,6 +252,7 @@ def get_instance(self, instance_id, **kwargs):
                 'tagReferences[id,tag[name,id]],'
                 'networkVlans[id,vlanNumber,networkSpace],'
                 'dedicatedHost.id,'
+                'monitoringServiceComponent,networkMonitors[queryType,lastResult,responseAction],'
```

Comment:
I think I would prefer you define this mask in the command itself, and send it in as an argument. The mask in here is getting fairly lengthy already and I would like to avoid growing it further.